src/app/settings/views/Scripts/ScriptsList/ScriptsList.test.tsx

```diff
@@ -85,84 +98,85 @@ describe("ScriptsList", () => {
   });
 
   it("Displays commissioning scripts by default", () => {
-    const store = mockStore(state);
-
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter initialEntries={[{ pathname: "/" }]}>
-          <CompatRouter>
-            <ScriptsList />
-          </CompatRouter>
-        </MemoryRouter>
-      </Provider>
+    renderWithMockStore(
+      <MemoryRouter initialEntries={[{ pathname: "/" }]}>
```

Comment:
It's possible these tests don't need the router (or at least I couldn't spot anything in the ScriptsList component).

Reply:
I tried removing the MemoryRouter from all of them, but then all of them failed, so I'm gonna keep them in there I reckon. They all failed with the same message: 
`TypeError: Cannot read properties of undefined (reading 'location')`